### PR TITLE
Add Platform GOP Policy

### PR DESCRIPTION
**Pull request by liujunming**
GOP driver has dependency on PlatformGopPolicy protocol,
especially the GetVbtData() interface.

Tracked-On: projectacrn/acrn-hypervisor#4365

Signed-off-by: Xiong Zhang <xiong.y.zhang@intel.com>
Signed-off-by: Junming Liu <junming.liu@intel.com>

### CODE REVIEW

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+#
+#
+##
+
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = PlatformGOPPolicy
+  FILE_GUID                      = 9737D7CA-D869-45e5-A5EF-75D9438688DE
```
Comment (peterfang) [inline OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: Please use a different GUID as this conflicts with Vlv2TbltDevicePkg/PlatformGopPolicy/PlatformGopPolicy.inf

**Review by peterfang — CHANGES_REQUESTED** [review]

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf (hunk 1 above)
Comment (liujunming) [reply OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: ok. I will use a different GUID.

**Comment by liujunming** [comment]
Hi, peter. The GUID has changed, please review it, thanks.

**Comment by liujunming** [comment]
@peterfang  Hi, peter. The GUID has changed, please review it, thanks.

**Review by peterfang — APPROVED** [review]

**Comment by peterfang** [comment]
Approved. Handing over to @yuchuyang for SDL review.

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#
+#
+# Copyright (c)  1999  - 2014, Intel Corporation. All rights reserved
```
Comment (peterfang) [inline OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: Can we update this to 2020?

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  BASE_NAME                      = PlatformGOPPolicy
+  FILE_GUID                      = 9737D7CA-D869-46e5-A5EF-75D9538688DE
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = PlatformGOPPolicyEntryPoint
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32
```
Comment (peterfang) [inline OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: This is just for reference but can we update this to X64? Or "IA32 X64"?

**Review by peterfang — CHANGES_REQUESTED** [review]
Just noticed a couple minor things I'd suggest changing. Thanks.

**Review by yuchuyang — APPROVED** [review]

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf (hunk 2 above)
Comment (liujunming) [reply OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: OK.

File: OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf (hunk 3 above)
Comment (liujunming) [reply OvmfPkg/PlatformGopPolicy/PlatformGopPolicy.inf]: > This is just for reference but can we update this to X64? Or "IA32 X64"?

ok.

**Comment by liujunming** [comment]
@peterfang  Hi,  peter. The minor changes have done. Please review it, thanks.

**Review by peterfang — APPROVED** [review]

**Pull request merged**